snap/system_usernames.go

```diff
@@ -155,4 +155,19 @@ var SupportedSystemUsernames = map[string]systemUsername{
 	"snap_microk8s": {Id: 584789, AllowedSnapIds: []string{
 		"EaXqgt1lyCaxKaQCU349mlodBkDCXRcg", // microk8s
 	}},
+	"snap_iotedge": {Id: 584790, AllowedSnapIds: []string{
```

Comment:
why is this not snap_aziotedge ?

Reply:
Because the azure-iot-edge codebase uses a different naming convention than the azure-iot-identity codebase (the one that uses the aziot prefixes). This is intentional and `snap_iotedge` is the correct name here but if this is absolutely not workable I can talk to the Microsoft folks